src/utils/wrappers.js

```diff
@@ -390,7 +390,6 @@ function handleWrappers (ast) {
                     parent.callee = replacement;
                   }
                   else if (parent.arguments && parent.arguments.some(arg => arg === node)) {
-                    delete parent.scope.declarations[node.id];
```

Comment:
We need a test for this code path because it must not have been tested or else we would have caught the bug

Reply:
Ok, added.